### DISCUSSION TITLE
Doc page using method that was moved

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#290] [Documentation]
I wasn't sure how I should post this, as an issue, a discussion or just somewhere on the Discord, so here it is 😄.
[This page](https://fish-networking.gitbook.io/docs/manual/guides/client-side-prediction/limitations-and-gotchas#triggers-and-collisions) mentions `TimeManager.IsReplaying` which was moved in 3.x to `PredictionManager.IsReplaying`.
Someone on the Discord had a problem locating the method as a result, and I figured the page should probably mention the new one.

**Comment by FirstGearGames** [comment]
Fixed btw and ty :)
